feature/system/throughput/otg_tests/dev_thruput/README.md

```diff
@@ -0,0 +1,180 @@
+A[ATE PORT1]<-IPv4-IPv6-B[DUT:PORT1] <-> C[DUT:PORT2] VLAN1
+C[DUT:PORT2] <-- IPv4-IPv6 --> D[PORT2:DUT];
+```
+VLAN1            |  VLAN2    | VLAN3     | VLAN4     | VLAN5      | VLAN6     |
+:----------------| :---------| :-------- | :---------| :----------| :---------|
+ATE1, DUT1, DUT2 | DUT3-DUT4 | DUT5-DUT6 | DUT7-DUT8 | DUT9-DUT10 | DUT11-DUT12
+
+VLAN13       |  VLAN14     | VLAN15      | VLAN16    | VLAN17       | VLAN18         |
+:------------| :-----------| :---------- | :---------| :------------| :--------------|
+DUT25, DUT26 | DUT27-DUT28 | DUT29-DUT30 | DUT31-DUT32 | DUT33-DUT34 | DUT35-DUT36-ATE2
+
+ATE-PORT1, DUT-PORT1 & DUT-PORT2 ---> VLAN1, 
+DUT-PORT3 & DUT-PORT4 ---> VLAN2,
+DUT-PORT5 & DUT-PORT6 ---> VLAN3,
+DUT-PORT7 & DUT-PORT8 ---> VLAN4,
+DUT-PORT9 & DUT-PORT10 ---> VLAN5,
+DUT-PORT11 & DUT-PORT12 ---> VLAN6,
+DUT-PORT13 & DUT-PORT14 ---> VLAN7,
+DUT-PORT15 & DUT-PORT16 ---> VLAN8,
+DUT-PORT17 & DUT-PORT18 ---> VLAN9,
+DUT-PORT19 & DUT-PORT20 ---> VLAN10,
+DUT-PORT21 & DUT-PORT22 ---> VLAN11,
+DUT-PORT23 & DUT-PORT24 ---> VLAN12,
+DUT-PORT25 & DUT-PORT26 ---> VLAN13,
+DUT-PORT27 & DUT-PORT28 ---> VLAN14
+DUT-PORT29 & DUT-PORT30 ---> VLAN15,
+DUT-PORT31 & DUT-PORT32 ---> VLAN16
+DUT-PORT33 & DUT-PORT34 ---> VLAN17
+DUT-PORT35 & DUT-PORT36 & ATE-PORT2 ---> VLAN18
+
+## Procedure
+
+### Testbed setup - Generate configuration for ATE and DUT
+
+  * Create 18 VLAN's on the DUT from VLAN1 to VLAN18 and are all configured as Access VLAN
+  * Assign IPv4, IPv6 addresses to ATE-PORT1, ATE-PORT2
+      ATE-PORT1 - IPv4 address 192.168.1.1/24; IPv6 address 2000:1:1:1::1/64
+      ATE-PORT1 - IPv4 address 193.168.1.1/24; IPv6 address 2001:1:1:1::1/64
+  * Traffic enters from ATE-PORT1 to DUT:PORT1 tagged/untagged and it is forwarded to 
+    DUT:PORT2 part of VLAN1. Then traffic is forwarded to DUT:PORT3 because it is 
+    back-back connected. Once it enters DUT:PORT3, traffic is forwarded to DUT:PORT4
+    and similar to the earlier scenario traffic enters into the DUT:PORT5 and follows
+    same till we reach the DUT-PORT36 of VLAN18 which is connected to ATE-PORT2
+  * Traffic entering from ATE-PORT2 follows similar forwarding mechanism as shown
+    in the previous step
+
+## Canonical OC
+```json
+{
+  "interfaces": {
+    "interface": [
+      {
+        "aggregation": {
+          "config": {
+            "lag-type": "LACP",
+            "min-links": 1
+          }
+        },
+        "config": {
+          "name": "ae0"
+        },
+        "name": "ae0"
+      },
+      {
+        "config": {
+          "loopback-mode": "FACILITY",
+          "name": "eth0"
+        },
+        "ethernet": {
+          "config": {
+            "aggregate-id": "ae0",
+            "duplex-mode": "FULL",
+            "port-speed": "SPEED_10GB"
+          }
+        },
+        "name": "eth0"
+      }
+    ]
+  }
+}
+```
+
+## TestCase-1:
+### PERF-1.1 Test IPv4 traffic 400G throughput
+  *   Test traffic using Fixed frame size of 64 bytes
```

Comment:
Need to also define number of packets (which IMO is preferred over duration) and a PPS rate.